build_tools/packaging/python/templates/rocm/setup.py

```diff
@@ -62,7 +62,7 @@ def import_dist_info():
 print("Found packages:", packages)
 
 setup(
-    name="rocm-sdk",
+    name="rocm",
```

Comment:
We should also update the docs at https://github.com/ROCm/TheRock/blob/main/RELEASES.md#installing-python-packages. Can wait until the first wheels are pushed though.